legal-api/src/legal_api/reports/business_document.py

```diff
@@ -464,8 +464,11 @@ def _set_amalgamation_details(self, business: dict):
 
     def _set_amalgamating_details(self, business: dict):
         amalgamating_info = business.get('business', {}).get('amalgamatedInto')
+        amalgamated_entity = business.get('business', {}).get('amalgamatedEntity')
         if amalgamating_info:
             business['business']['isAmalgamating'] = True
+            if amalgamated_entity:
```

Comment:
add this so the TING business will not go through the `Amalgamations.html`

Reply:
I don't think its correct. https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/business_document.py#L436 this will return none for Amalgamating business (unless this business created through amalgamation).